tests/test_pinky.rs

```diff
@@ -36,14 +36,32 @@ fn test_long_format() {
 
 #[cfg(target_os = "linux")]
 #[test]
-fn test_short_format() {
-    let scene = TestScenario::new(util_name!());
-
+fn test_short_format_i() {
+    // allow whitespace variation
+    // * minor whitespace differences occur between platform built-in outputs; specifically, the number of trailing TABs may be variant
```

Comment:
When you say this, are you referring to variations between GNU coreutils on different platforms, or do you mean the default versions of `pinky` installed on different platforms (which may or may not be from GNU coreutils)?